src/FFmpegWriter.cpp

```diff
@@ -2019,10 +2084,19 @@ void FFmpegWriter::InitScalers(int source_width, int source_height) {
 		} else
 #endif
 		{
-			img_convert_ctx = sws_getContext(source_width, source_height, PIX_FMT_RGBA, info.width, info.height, AV_GET_CODEC_PIXEL_FORMAT(video_st, video_st->codec), SWS_BILINEAR,
+			img_convert_ctx = sws_getContext(source_width, source_height, PIX_FMT_RGBA, info.width, info.height, video_codec->pix_fmt, SWS_BILINEAR,
```

Comment:
Same thing again: `AV_GET_CODEC_PIXEL_FORMAT(video_st, video_st->codec)` instead of pix_fmt